app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -109,7 +110,7 @@ fun WallpaperSettings(
 @Composable
 private fun WallpaperGroupHeading(
```

Comment:
nit: should this function have some kdocs?

Reply:
It's `private` so kDoc aren't strictly enforced by ktlint but I think we can always encourage kDocs for more complicated functions

Reply:
I would also not expect to address it here, since this PR doesn't really impact it. I could address it in a follow-up if you'd like, though

Reply:
I don't think it's super important right now.